contactsummary/CRM/Contactsummary/Utils/Staff.php

```diff
@@ -2,25 +2,27 @@
 
 class CRM_Contactsummary_Utils_Staff {
   /**
-   * Get the total number of staff.
+   * Get the total number of staff (Individual contacts with active contracts).
    *
    * @return int
    */
   public static function getStaffNum() {
-    $query = "
-    SELECT COUNT(DISTINCT c.id) count
-    FROM civicrm_contact c
-
-    INNER JOIN civicrm_hrjobcontract jc
-    ON (c.id = jc.contact_id AND jc.is_primary = 1)
 
-    INNER JOIN civicrm_hrjobcontract_revision jcr
-    ON (jc.id = jcr.jobcontract_id AND jcr.effective_date <= NOW())
+    $currentDate = date('Y-m-d');
 
-    INNER JOIN civicrm_hrjobcontract_details jcd
-    ON (jcr.id = jcd.jobcontract_revision_id AND (jcd.period_end_date >= NOW() OR jcd.period_end_date IS NULL))
-
-    WHERE c.contact_type = 'Individual'";
+    $query = "
```

Comment:
I know this method was here before and all you did was fixing it, but I think all these details about the job contract tables and structure should be hidden under HRJobContract BAOs. Also, have you confirmed what is considered to be an active contract (Remember that discussion we started last week on hipchat?)?